downstream/modules/eda/proc-eda-set-up-token.adoc

```diff
@@ -24,7 +24,7 @@ For more information about creating the token, see the link:https://docs.ansible
 +
 [NOTE]
 ====
-The token must be in write-scope.
+Your token must be in write-scope and if you are the assigned user, you must have permission to execute job and workflow templates.
 ====
 . Select btn:[Create controller token].
```

Comment:
There should be a .Procedure before the steps.

Reply:
Hi @dcdacosta. This peer review was or the note only. There's already a .Procedure before the steps on line 11 above if you expand the section. Is that want you mean?